bin/templates/project/__PROJECT_NAME__/config.xml

```diff
@@ -65,6 +65,10 @@
     <feature name="LocalStorage">
         <param name="ios-package" value="CDVLocalStorage"/>
     </feature>
+    <feature name="Console">
```

Comment:
You need to add this in defaults.xml as well. I tried adding the platform to a newly created CLI project, console.log failed because this wasn't in config.xml.

See https://github.com/apache/cordova-ios/commit/b0c71bf63c8ce989648f0b8205befe174af37860